### PR TITLE
Cloud Changelog - Jan 19

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -3,12 +3,62 @@
...
+#### Administration
+ - Plugins are re-initialized only when Plugin related config changes happen. This improves performance by avoiding plugin initialization which is quite costly.
+ - Added server support for receiving binary (messagepack encoded) websocket messages.
+ - Removed dead struct ``ManifestExecutables``.
+ - Added new flag ``showTeamSidebar`` in ``registerProducts``, which, when set to ``true``, displays the team sidebar in the product.
+ - Increased key length in plugin KV store to 150.
+ - Admins are now able to upgrade the server to the enterprise version and request the trial license with a single button click enhancing product adoption by an improved user experience.
+ - Added support for exporting and importing the type and ``edit_at`` of a post.
+ - Added support for ``postgresql`` schema designator.
+ - A new schema migrations system was added.
```
Comment (amyblais) [inline source/install/cloud-changelog.md:37]: Let me know if some / none of these are not needed for cloud changelog.

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -3,12 +3,62 @@
...
 - [Release 2021-11-23](#release-2021-11-23)
 - [Release 2021-11-11](#release-2021-11-11)
-- [Release 2021-11-10](#release-2021-11-10)
+
+## Release 2022-01-19
+
+### Improvements
+
+#### User Interface (UI)
+ - The Renew Now button is no longer shown if the license id does not exist in the portal.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:18]:
```suggestion
 - The **Renew Now** button is no longer shown if the license id does not exist in the portal. Instead, **Contact Sales** is shown.
```

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -3,12 +3,62 @@
...
+ - Updated the About Mattermost Team Edition modal to change the community link from `mattermost.org` to `mattermost.com/community/`.
+ - Invite to team modal now auto-focuses its email search input.
+ - Added a new dialog for Remove License confirmation.
+ - Updated "Terms of Service" terminology to "Terms of Use" product-wide.
+ - Updated **Account Settings** terminology to **Settings**.
+ - Added Accept-Language header to generate link previews in the Default Server language.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:24]:
```suggestion
 - Added Accept-Language header to generate link previews in the default Server language.
```

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -3,12 +3,62 @@
...
+ - Added a new modal for the upload license workflow in the **System Console > Edition and License** page.
+ - The "Invite Members" button is now hidden when the Direct Message category is collapsed.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:26]:
```suggestion
 - The **Invite Members** button is now hidden when the Direct Message category is collapsed.
```

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -3,12 +3,62 @@
...
+
+#### Administration
+ - Plugins are re-initialized only when Plugin related config changes happen. This improves performance by avoiding plugin initialization which is quite costly.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:29]:
```suggestion
 - Plugins are re-initialized only when Plugin-related configuration changes happen. This improves performance by avoiding costly plugin initialization.
```

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -3,12 +3,62 @@
...
+ - Added server support for receiving binary (messagepack encoded) websocket messages.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:30]:
```suggestion
 - Added server support for receiving binary (messagepack-encoded) websocket messages.
```

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -3,12 +3,62 @@
...
+ - Removed dead struct ``ManifestExecutables``.
+ - Added new flag ``showTeamSidebar`` in ``registerProducts``, which, when set to ``true``, displays the team sidebar in the product.
+ - Increased key length in plugin KV store to 150.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:33]: It is generally understood what a plugin KV store is?

File: source/install/cloud-changelog.md [hunk 8]
```diff
@@ -3,12 +3,62 @@
...
 - [Release 2021-12-01](#release-2021-12-01)
 - [Release 2021-11-23](#release-2021-11-23)
 - [Release 2021-11-11](#release-2021-11-11)
-- [Release 2021-11-10](#release-2021-11-10)
+
+## Release 2022-01-19
+
+### Improvements
+
+#### User Interface (UI)
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:17]: - Invite to team modal now auto-focuses its email search input.
 - Updated "Terms of Service" terminology to "Terms of Use" product-wide.
 - Updated **Account Settings** terminology to **Settings**.
 - Added Accept-Language header to generate link previews in the Default Server language.
 - The "Invite Members" button is now hidden when the Direct Message category is collapsed.

Comment (jasonblais) [reply source/install/cloud-changelog.md:17]: Removed items that do not impact Cloud

File: source/install/cloud-changelog.md (hunk 1 above)
Comment (jasonblais) [reply source/install/cloud-changelog.md:37]: I don't know the details but they all seem specific to self-hosted admins.

Perhaps the first bullet point would be if we rephrase it to

```
Improved plugins performance by re-initializing only when Plugin related config changes happen.
```

File: source/install/cloud-changelog.md [hunk 9]
```diff
@@ -3,12 +3,62 @@
...
+ - Removed dead struct ``ManifestExecutables``.
+ - Added new flag ``showTeamSidebar`` in ``registerProducts``, which, when set to ``true``, displays the team sidebar in the product.
+ - Increased key length in plugin KV store to 150.
+ - Admins are now able to upgrade the server to the enterprise version and request the trial license with a single button click enhancing product adoption by an improved user experience.
+ - Added support for exporting and importing the type and ``edit_at`` of a post.
+ - Added support for ``postgresql`` schema designator.
+ - A new schema migrations system was added.
+
+### Bug Fixes
+ - Fixed an issue where when clicking the Upgrade to Enterprise Edition button, the upgrade progress bar and the restart button were no longer shown once the progress reached 100%.
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:40]: Doesn't impact Cloud

Comment (jasonblais) [reply source/install/cloud-changelog.md:40]: Generally any related to upgrades or licensing does not apply to Cloud

**Review by jasonblais — CHANGES_REQUESTED** [review]

File: source/install/cloud-changelog.md [hunk 10]
```diff
@@ -3,12 +3,62 @@
...
+ - Added a new modal for the upload license workflow in the **System Console > Edition and License** page.
+ - The "Invite Members" button is now hidden when the Direct Message category is collapsed.
+
+#### Administration
+ - Plugins are re-initialized only when Plugin related config changes happen. This improves performance by avoiding plugin initialization which is quite costly.
+ - Added server support for receiving binary (messagepack encoded) websocket messages.
+ - Removed dead struct ``ManifestExecutables``.
+ - Added new flag ``showTeamSidebar`` in ``registerProducts``, which, when set to ``true``, displays the team sidebar in the product.
+ - Increased key length in plugin KV store to 150.
+ - Admins are now able to upgrade the server to the enterprise version and request the trial license with a single button click enhancing product adoption by an improved user experience.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:34]:
```suggestion
 - System Admins are now able to upgrade the server to the Enterprise edition and request the trial license with a single click for a simplified user experience.
```

File: source/install/cloud-changelog.md [hunk 11]
```diff
@@ -3,12 +3,62 @@
...
+ - Added support for exporting and importing the type and ``edit_at`` of a post.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:35]:
```suggestion
 - Added support for exporting and importing the post type and ``edit_at`` post details.
```

Comment (cwarnermm) [reply source/install/cloud-changelog.md:35]: It isn't clear what "edit_at" post data refers to specifically. Is it possible to elaborate a bit further on this one to communicate the value of the change to users?

File: source/install/cloud-changelog.md (hunk 9 above)
Comment (cwarnermm) [inline source/install/cloud-changelog.md:40]:
```suggestion
 - Fixed an issue where when selecting the **Upgrade to Enterprise Edition** button. The upgrade progress bar and the **Restart** button were no longer shown once the progress reached 100%.
```

File: source/install/cloud-changelog.md [hunk 12]
```diff
@@ -3,12 +3,62 @@
...
+ - Added support for ``postgresql`` schema designator.
+ - A new schema migrations system was added.
+
+### Bug Fixes
+ - Fixed an issue where when clicking the Upgrade to Enterprise Edition button, the upgrade progress bar and the restart button were no longer shown once the progress reached 100%.
+ - Fixed an issue where the user avatar wasn’t removed from the participants list after the user’s post was deleted and they had no other posts in the thread.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:41]:
```suggestion
 - Fixed an issue where the user avatar wasn’t removed from the participants list after the user’s only post in a thread was deleted.
```

**Review by cwarnermm — COMMENTED** [review]
Feedback provided inline. Thanks!

**Review by marianunez — APPROVED** [review]

File: source/install/cloud-changelog.md (hunk 1 above)
Comment (agnivade) [reply source/install/cloud-changelog.md]: Yes, it's mainly pertitent to self-hosted admins. We will provide an exhaustive note during the on-prem release.

**Review by agnivade — APPROVED** [review]

**Review by isacikgoz — APPROVED** [review]

**Review by jasonblais — APPROVED** [review]

**Review by esethna — APPROVED** [review]
